src/components/dock/dock.scss

```diff
@@ -70,7 +70,7 @@ nav {
     flex-grow: 1;
 
     padding: var(--dock-padding); // needed for the focus effect
-
+    padding-bottom: calc((var(--dock-padding) * 2.4));
```

Comment:
I think this is unrelated change. Could you please remove this commit.

Reply:
⚡